docs/how-tos/manage-fleet.md

```diff
@@ -25,69 +25,155 @@ For example, you might have 30 robots in one warehouse and 500 in another.
-You can use {{< glossary_tooltip term_id="fragment" text="fragments" >}} to streamline the process of configuring multiple similar machines.
+Click on the organization dropdown in the top navigation bar and click on **Settings** to create and manage API keys.
 
-{{<imgproc src="/fleet/app-usage/create-machine.png" class="fill aligncenter" resize="800x" style="max-width: 600px" declaredimensions=true alt="Create a new machine in the Viam app">}}
+{{<imgproc src="/cloud/rbac.png" resize="700x" declaredimensions=true alt="Organization page" class="aligncenter">}}
 
 {{% /tablestep %}}
-{{% tablestep %}}
-**5. Monitor your fleet**
+{{< /table >}}
 
-Using the [Viam app](https://app.viam.com), you can:
+## Monitoring and Debugging
 
-- Monitor the status of each machine from its [**LOGS** tab](/cloud/machines/#logs).
-- View any data captured by your fleet from the [**Data** tab](/services/data/).
-- Operate machines remotely from the [**CONTROL** tab](/fleet/control/).
+Viam provides you with the ability to access each machine, check its logs, check recent changes, and roll back changes if needed.
+
+{{< table >}}
+{{% tablestep link="/cloud/machines/#logs" %}}
+**1. Monitor your fleet's logs**
+
+Using the [Viam app](https://app.viam.com), you can monitor the status of each machine from its [**LOGS** tab](/cloud/machines/#logs).
+
+{{<gif webm_src="/fleet/log-filtering.webm" mp4_src="/fleet/log-filtering.mp4" alt="Filter logs by term of log level in the UI" max-width="800px">}}
+
+You can also access machine logs using [`viam machines logs`](/cli/#machines-alias-robots), the [Machines API](/appendix/apis/robot/), or the [Viam mobile app](/cloud/machines/#logs).
 
 {{% /tablestep %}}
-{{< /table >}}
+{{% tablestep link="/fleet/control/" %}}
+**2. Test your machines remotely**
 
-## Next steps
+Using the [Viam app](https://app.viam.com), you can remotely operate machines from the [**CONTROL** tab](/fleet/control/).
 
-{{< cards >}}
-{{% card link="/fleet/" %}}
-{{% card link="/cli/" %}}
-{{% card link="/cloud/rbac/" %}}
+{{<gif webm_src="/fleet/control.webm" mp4_src="/fleet/control.mp4" alt="Using the control tab" max-width="800px">}}
+
+You can also operate machine using the [`viam machines part run`](/cli/#machines-alias-robots) or [Viam's APIs](/appendix/apis/), or the [Viam mobile app](/fleet/control/#control-interface-in-the-viam-mobile-app).
 
-<!-- markdownlint-disable MD034 -->
+{{% /tablestep %}}
+{{% tablestep link="/configure/#configuration-history" %}}
+**3. Use configuration version history**
 
-{{% manualcard link="https://www.viam.com/post/integrate-your-ros2-robot-with-viam/" target="_blank" %}}
+The Viam app keeps a record of your configuration changes, allowing you to revert to earlier configurations if needed. To see the history of the configuration of a machine part, click on **History** on the component or machine menu on the CONFIGURE tab.
```

Comment:
```suggestion
The Viam app keeps a record of your configuration changes, allowing you to revert to earlier configurations if needed.
To see the history of the configuration of a machine part, click on **History** on the **CONFIGURE** tab.
```
I don't see this on the component and I don't think people will know what "machine menu" means, so we can simplify. Also newline nit